### PR TITLE
[docs] Add status header for parquet exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for parquet exporter

### CODE REVIEW

File: exporter/parquetexporter/README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # Parquet File Exporter
 
-Sends pipeline data to Parquet files.
+| Status                   |                      |
+| ------------------------ |----------------------|
+| Stability                | [beta]               |
```
Comment (pmm-sumo) [inline exporter/parquetexporter/README.md:5]:
```suggestion
| Stability                | [in development]               |
```

File: exporter/parquetexporter/README.md [hunk 2]
```diff
@@ -25,4 +29,7 @@ exporters:
 ```
 
 The full list of settings exposed for this exporter are documented [here](config.go)
-with detailed sample configurations [here](testdata/config.yaml).
\ No newline at end of file
+with detailed sample configurations [here](testdata/config.yaml).
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```
Comment (pmm-sumo) [inline exporter/parquetexporter/README.md:34]:
```suggestion
[in-development]:https://github.com/open-telemetry/opentelemetry-collector#in-development
```

**Review by pmm-sumo — CHANGES_REQUESTED** [review]
This one is not even functional, we should use "in-development" status

File: exporter/parquetexporter/README.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
 # Parquet File Exporter
 
-Sends pipeline data to Parquet files.
+| Status                   |                      |
+| ------------------------ |----------------------|
+| Stability                | [in-development]     |
+| Supported pipeline types | trace, logs, metrics |
+| Distributions            | [contrib]            |
```
Comment (codeboten) [inline exporter/parquetexporter/README.md:7]: Currently this component is not in contrib

```suggestion
| Distributions            | none                 |
```

File: exporter/parquetexporter/README.md [hunk 4]
```diff
@@ -25,4 +29,6 @@ exporters:
 ```
 
 The full list of settings exposed for this exporter are documented [here](config.go)
```
Comment (pmm-sumo) [inline exporter/parquetexporter/README.md:31]: This is a dead link, I suggest to fix it. Perhaps using that way:

```suggestion
The full list of settings exposed for this exporter is going to be documented later
```

**Review by pmm-sumo — APPROVED** [review]
The change looks good overall

**Review by codeboten — APPROVED** [review]

**Pull request merged**